02_Development/Display/Tests/test_spl06_007.py

```diff
@@ -0,0 +1,355 @@
+                        "and pressure.")
+
+    def test_set_sampling_valid_values(self):
+        self.assertTrue(
+            self._sensor.set_sampling(pressure_oversample=1,
+                                      pressure_sampling_rate=1,
+                                      temperature_oversample=1,
+                                      temperature_sampling_rate=1),
+            "Fails to set the oversample and sampling rate "
+            "to valid custom values for temperature and "
+            "pressure")
+
+    def test_set_sampling_invalid_values(self):
+        with self.assertRaises(ValueError,
+                               msg="Fails to raise a ValueError when "
+                               "oversample or temperature values are not "
+                               "in the set {1, 2, 4, 8, 16, 32, 64, 128}."):
+            self._sensor.set_sampling(pressure_oversample=3,
+                                      pressure_sampling_rate=3,
+                                      temperature_oversample=3,
+                                      temperature_sampling_rate=3)
+
+    def test_pressure_without_sampling_set(self):
+        self.assertTrue(math.isnan(self._sensor.pressure()),
+                        "Fails to return NaN for pressure when the user "
+                        "has not yet set the sampling parameters.")
+        self.assertTrue(math.isnan(self._sensor.temperature()),
+                        "Fails to return NaN for temperature when the "
+                        "user has not yet set the sampling parameters.")
+
+    @unittest.skipIf(not is_on_raspberry_pi(),
+                     "Cannot determine ambient temperature unless "
+                     "connected to hardware.")
+    def test_ambient_temperature(self):
+        self._sensor.set_op_mode(PressureSensor.OpMode.command)
+        self._sensor.set_sampling(temperature_sampling_rate=8)
+        measured_temperature = self._sensor.temperature()
+        self.assertTrue(math.isclose(measured_temperature, 20,
+                                     rel_tol=0.50),
+                        f"{measured_temperature} != "
+                        "20 +/- 50% degC :\n"
+                        "Fails to return ambient temperature in "
+                        "degC.\nNote that if this test is "
+                        "performed in a very cold or hot "
+                        "environment, the\nambient temperature "
+                        "may fall outside the range of this test.")
+
+    @unittest.skipIf(not is_on_raspberry_pi(),
+                     "Cannot determine ambient pressure unless "
+                     "connected to hardware.")
+    def test_ambient_pressure(self):
+        self._sensor.set_sampling()
+        self._sensor.set_op_mode(PressureSensor.OpMode.command)
+        measured_pressure = self._sensor.pressure()
+        self.assertTrue(math.isclose(measured_pressure, 101325,
```

Comment:
`101325` is a pretty magical number.  Is that supposed to be sea level or something?  (If it is, what happens if someone tests on a mountain?)